lib/enumerize/attribute.rb

```diff
@@ -13,6 +13,7 @@ def initialize(klass, name, options={})
       @klass  = klass
       @name   = name.to_sym
       @i18n_scope = options[:i18n_scope]
+      @origin_options = options
```

Comment:
could you name it simply `options` or is there any meaning behind `origin_options` name?

Reply:
@nashby Thank you for your reply.

The options method already exists in `Enumerize::Attribute` and I thought it appropriate to use an alias to maintain compatibility and avoid conflicts.

Since options is the name used for the enumerize method, I followed it.

For the above reasons, we named it `origin_options` to indicate that it is an option specified by enumerize.

I wasn't sure whether to use `original_options` or not, but I decided to use `origin_options` because I wanted to make it shorter.

Reply:
The name options is already in use, and we needed to avoid the name options to maintain compatibility. Therefore, we propose "arguments" as a simple name. Then we pushed.